src/mesos_state_client.erl

```diff
@@ -28,6 +29,7 @@ proto() ->
       "http"
```

Comment:
nothing, just adds spec since I was going down all of the new functions with them